src/ctia/entity/note/schemas.clj

```diff
@@ -47,11 +49,13 @@
 (def searchable-fields
   #{:id
     :source
-    :entity_id
+    :related_entities.entity_id
+    :related_entities.entity_type
+    :note_class
```

Comment:
searchable fields are for fields that we want to sort on by default, without actively filtering on it. I do not think that's desired here.

Reply:
I removed the unneeded fields